std/shared/src/main/scala/collection/Set.scala

```diff
@@ -1,11 +1,15 @@
 package scalaz
 package std
 
-import typeclass.EqClass
+import typeclass.{ EqClass, FunctorClass }
 
 trait SetInstances {
   implicit def setEq[A: Eq]: Eq[Set[A]] =
     instanceOf(new EqClass[Set[A]] {
       def equal(first: Set[A], second: Set[A]): Boolean = (first.toStream.corresponds(second.toStream)(Eq[A].equal))
     })
+
+  implicit val setFunctor: Functor[Set] = instanceOf(new FunctorClass[Set] {
+    def map[A, B](ma: Set[A])(f: A => B): Set[B] = ma.map(f)
+  })
```

Comment:
this has similar problems to `Monad[Set]`

Reply:
Ah, you are right!